src/main/java/org/akaza/openclinica/controller/helper/TemplateHelper.java

```diff
@@ -0,0 +1,36 @@
+package org.akaza.openclinica.controller.helper;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Random;
+
+@JsonIgnoreProperties
+public class TemplateHelper {
+
+
+    public Integer random(int digit_number) {
+        switch (digit_number) {
+            case 1:
+                return new Random().nextInt(9);
```

Comment:
apparently it is better to use ThreadLocalRandom.current().nextInt(min, max + 1).
see: https://stackoverflow.com/questions/363681/how-do-i-generate-random-integers-within-a-specific-range-in-java